frontend/src/features/mapSideBar/SidebarContents/BuildingForm.tsx

```diff
@@ -186,12 +236,45 @@ interface IBuildingForm {
   setMovingPinNameSpace: (nameSpace: string) => void;
   /** to help determine the namespace of the field (eg. address.line1) */
   nameSpace: string;
-  /** Notify the parent that the building has been saved, potentially starting a new workflow. */
-  setBuildingToAssociateLand: (building: IBuilding) => void;
+  /** Go to the associated land form directly */
+  goToAssociatedLand: (building: IBuilding) => void;
```

Comment:
this is for the modify associated land button, which doesn't need to go to the generate associated land page.